server/commands/check.go

```diff
@@ -83,6 +83,11 @@ func (query *CheckQuery) Execute(ctx context.Context, req *openfgapb.CheckReques
 	if err != nil {
 		return nil, err
 	}
+	if userset == nil {
```

Comment:
Why not just add
```
objectType, _ := tupleUtils.SplitObject(object)

_, err = typesys.GetRelation(objectType, relation)
if err != nil {
    continue
}
```
below [this line of code](https://github.com/openfga/openfga/pull/342/files#diff-9f6cc2751e9bcd87e5d066d5d601ded4d87e40818657f670bc0b6d84816c63c6R272) in the `resolveDirectUserset` function?

The suggestion above makes the behavior way more obvious that if you are reading userset values from the database and the userset you received from the database doesn't validate against the model then it is skipped with the `continue` line. The way this code is written is not obvious that we're skipping userset values with an undefined objectType or relation (according to the current model), and that's because the code isn't co-located with the immediate body of code where we are reading those kind of values from the database.

Reply:
Good idea, but as discussed offline, i can't remove this block of code, only the `if nsUS == nil { return nil }` block.